cmake/fake_tribits.cmake

```diff
@@ -34,88 +32,59 @@ MACRO(ADD_INTERFACE_LIBRARY LIB_NAME)
 ENDMACRO()
 
 FUNCTION(KOKKOS_ADD_TEST)
-  if (KOKKOS_HAS_TRILINOS)
-    CMAKE_PARSE_ARGUMENTS(TEST
-      "SKIP_TRIBITS"
-      "EXE;NAME;TOOL"
-      "ARGS"
-      ${ARGN})
-
-    IF(TEST_SKIP_TRIBITS)
-      MESSAGE(STATUS "Skipping test ${TEST_NAME} in TriBits")
-      RETURN()
-    ENDIF()
-
-    IF(TEST_EXE)
-      SET(EXE_ROOT ${TEST_EXE})
-    ELSE()
-      SET(EXE_ROOT ${TEST_NAME})
-    ENDIF()
-
-    TRIBITS_ADD_TEST(
```

Comment:
@bartlettroscoe just to confirm the changes are not intended to remove testing of kokkos within Trilinos, I've been able to build the kokkos tests in Trilinos with the changes in this PR, and the AT2 jobs also built the kokkos tests, though maybe that is unexpected since `tribits_add_test` is not used?
It's not clear to me why the tests are not building as expected in the AT1 jobs (there is some issue with which version or how gtest was found)